### ISSUE TITLE
`precision: 0` breaks numerical input

### ISSUE THREAD

**Issue by nandorojo** [CaioQuirinoMedeiros/react-native-mask-input#10]
```tsx
createNumberMask({
  prefix: ['$'],
  delimiter: ',',
  separator: '.',
  precision: 0
})
```

I'm trying to use this for a mask, so that you can only use integer dollar values like `$200`, but it ends up prepending a decimal:

<img width="116" alt="Screen Shot 2021-11-18 at 5 42 09 PM" src="https://user-images.githubusercontent.com/13172299/142508533-67579d11-18c7-4cc7-a377-a86f0b11a210.png">

And if you enter 4 digits, it's totally off:

<img width="278" alt="Screen Shot 2021-11-18 at 5 45 24 PM" src="https://user-images.githubusercontent.com/13172299/142508742-0482e394-b6d9-45e3-a292-fb18e2f91606.png">

Any suggestions on how to get around this @CaioQuirinoMedeiros?

Snack: https://snack.expo.dev/@nandorojo/humiliated-pretzel

Full code here:

```js
import MaskInput, { createNumberMask } from 'react-native-mask-input';
import React from 'react'

const dollarMask = createNumberMask({
  prefix: ['$'],
  delimiter: ',',
  separator: '.',
  precision: 0,
})

export default function MyComponent() {
  const [value, setValue] = React.useState('');

  return (
    <MaskInput
      value={value}
      mask={dollarMask}
      onChangeText={(masked, unmasked) => {
        setValue(unmasked); // you can use the masked value as well

        // assuming you typed "123456":
        console.log(masked); // "U$ 1,234.56"
        console.log(unmasked); // "123456"
      }}
      style={{ fontSize: 40}}
      autoFocus
    />
  );
}
```

**Comment by nandorojo** [comment]
This works fine on the current input library FWIW

**Comment by CaioQuirinoMedeiros** [comment]
Good to know... anyway, I just got it fixed on release v1.1.0

**Comment by aprilmintacpineda** [comment]
Still doesn't work on v1.1.0
If I put in `1000`, it shows `,1000` instead of `1,000`

![image](https://user-images.githubusercontent.com/21032419/146385197-e19cc9d2-5e9e-4b49-a976-4a2fb189c03b.png)

Can this be reopened?

```tsx
import MaskInput, { createNumberMask } from 'react-native-mask-input';
import React from 'react'

const dollarMask = createNumberMask({
  delimiter: ',',
  separator: '.',
  precision: 0
})

export default function MyComponent() {
  const [value, setValue] = React.useState('');

  return (
    <MaskInput
      value={value}
      mask={dollarMask}
      onChangeText={(masked, unmasked) => {
        setValue(unmasked); // you can use the masked value as well

        // assuming you typed "123456":
        console.log(masked); // "U$ 1,234.56"
        console.log(unmasked); // "123456"
      }}
      style={{ fontSize: 40}}
      autoFocus
    />
  );
}
```

https://snack.expo.dev/@nandorojo/humiliated-pretzel

**Comment by CaioQuirinoMedeiros** [comment]
@aprilmintacpineda fixed on v1.1.1
